drivers/display/display_ili9340.c

```diff
@@ -6,9 +6,10 @@
 #include "display_ili9340.h"
 #include <drivers/display/ili9340.h>
 
-#define SYS_LOG_DOMAIN "ILI9340"
-#define SYS_LOG_LEVEL CONFIG_SYS_LOG_ILI9340_LEVEL
-#include <logging/sys_log.h>
+#define LOG_MODULE_NAME display
+#define LOG_LEVEL CONFIG_LOG_ILI9340_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER();
```

Comment:
This has recently changed, so LOG_MODULE_REGISTER() now takes the LOG_MODULE_NAME, and you don't need to explicit set LOG_MODULE_NAME.  You can just do:

`LOG_MODULE_REGISTER(foo);`

Reply:
noticed it, fixed